roles/cs.aws-loadbalancer/tasks/main.yml

```diff
@@ -4,7 +4,7 @@
     region: "{{ aws_region }}"
     name: "{{ aws_elb_name }}"
     subnets: "{{ aws_vpc_subnet_ids }}"
-    aws_security_group_names:
+    security_group_names:
```

Comment:
FYI This has been renamed in some recent version that provides the `ec2_elb_lb` recipe.